bip-0442.md

```diff
@@ -90,138 +91,187 @@ highly optimized for just 2 stack elements.
+{
+    return (HashWriter{HASHER_PAIRCOMMIT} << x1 << x2).GetSHA256();
+}
+```
 
-### Behaviors LNhance tries to avoid introducing
+### PR
 
-The following behaviors are out of scope for LNhance and should not be enabled
-as a side effect without explicit consensus:
+https://github.com/lnhance/bitcoin/pull/6/files
 
-* Fine-grained introspection
-* State-carrying covenants
-* Bigint operations
-* New arithmetic capabilities using lookup tables
+## Rationale
 
-### Alternative approaches
+### Cost comparison of Lightning Symmetry constructions
 
-The following list of alternative approaches were discussed and rejected for
-various reasons, either for expanding the scope or for unnecessary complexity:
+The following table briefly summarizes the costs associated with force closing
+Lightning Symmetry channels enabled by various combinations of proposed script
+upgrades.
 
-* OP_CAT
-* SHA256 streaming opcodes
-* Merkle operation opcodes
-* 'Kitty' CAT: result or inputs arbitrarily limited in size
-* OP_CHECKTEMPLATEVERIFY committing to the taproot annex in tapscript
-* OP_CHECKSIGFROMSTACK on n elements as message
-* OP_VECTORCOMMIT: generalized form for n > 2 elements
-* ReKey: key delegation and multiple use of OP_CHECKSIGFROMSTACK
-
-### Cost comparison of LN-Symmetry constructions
-
-| Method        | ChannelSc | UpdateSc | UpdateW | ForceC  | Contest | Settle |
-| :------------ | --------: | -------: | ------: | ------: | ------: | :----: |
-| APO-Annex     |      8 WU |   113 WU |  100 WU | 1221 WU |  627 WU | SigOp  |
-| APO-Return    |      8 WU |   113 WU |   66 WU | 1359 WU |  765 WU | SigOp  |
-| CTV+CSFS+IKEY |     10 WU |    48 WU |   98 WU | 1328 WU |  732 WU |  CTV   |
-| CTV+CSFS      |     43 WU |    81 WU |   98 WU | 1394 WU |  765 WU |  CTV   |
-| LNhance       |     11 WU |    49 WU |  131 WU | 1191 WU |  594 WU |  CTV   |
+| Method           | ChannelSc | UpdateSc | UpdateW | ForceC  | Contest | Settle |
+| :--------------- | --------: | -------: | ------: | ------: | ------: | :----: |
+| APO-Annex        |      8 WU |   113 WU |  100 WU | 1221 WU |  627 WU | SigOp  |
+| APO-Return       |      8 WU |   113 WU |   66 WU | 1359 WU |  765 WU | SigOp  |
+| CTV+CSFS         |     43 WU |    81 WU |   98 WU | 1394 WU |  765 WU |  CTV   |
+| CTV+CSFS+IKEY    |     10 WU |    48 WU |   98 WU | 1328 WU |  732 WU |  CTV   |
+| CTV+CSFS+IKEY+PC |     11 WU |    49 WU |  131 WU | 1191 WU |  594 WU |  CTV   |
 
 *ChannelSc: channel script, UpdateSc: update script, UpdateW: witness is the
-same size for both Force Close and Contest in LN-Symmetry, ForceC: total cost of unilateral close transactions*
+same size for both Force Close and Contest in Lightning Symmetry, ForceC: total
+cost of unilateral close transactions, Contest: The additional cost to contest
+a force closure, Settle: Whether a signature operation or CTV operation is
+required to validate the settlement transaction*
```

Comment:
Thanks for expanding on the column titles. As I wrote above, it would be good to provide more context for this table. It would especially help if the compared schemata were at least briefly described and/or sources for details were provided.